### PR TITLE
added uvec creation

### PR DESCRIPTION
**Pull request by aronnoordam**

### CODE REVIEW

File: benchmark_tests/test_train_uvec_3d/test_train_on_embankment_3d.py [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+from stem.soil_material import OnePhaseSoil, LinearElasticSoil, SoilMaterial, SaturatedLaw
+from stem.load import UvecLoad
+from stem.boundary import DisplacementConstraint
+from stem.solver import AnalysisType, SolutionType, TimeIntegration, DisplacementConvergenceCriteria, StressInitialisationType, SolverSettings, Problem
+from stem.output import NodalOutput, VtkOutputParameters, Output
+from stem.stem import Stem
+from shutil import rmtree, copytree
+
+import sys
+sys.path.append("./")
```
Comment (aronnoordam) [inline benchmark_tests/test_train_uvec_3d/test_train_on_embankment_3d.py:12]:
```suggestion
```

File: benchmark_tests/test_train_uvec_3d/test_train_on_embankment_3d.py [hunk 2]
```diff
@@ -0,0 +1,157 @@
+import os
+from stem.model import Model
+from stem.soil_material import OnePhaseSoil, LinearElasticSoil, SoilMaterial, SaturatedLaw
+from stem.load import UvecLoad
+from stem.boundary import DisplacementConstraint
+from stem.solver import AnalysisType, SolutionType, TimeIntegration, DisplacementConvergenceCriteria, StressInitialisationType, SolverSettings, Problem
+from stem.output import NodalOutput, VtkOutputParameters, Output
+from stem.stem import Stem
+from shutil import rmtree, copytree
```
Comment (aronnoordam) [inline benchmark_tests/test_train_uvec_3d/test_train_on_embankment_3d.py:9]: shutil  up

File: benchmark_tests/test_train_uvec_3d/uvec_ten_dof_vehicle_2D/__init__.py
Comment (aronnoordam) [inline benchmark_tests/test_train_uvec_3d/uvec_ten_dof_vehicle_2D/__init__.py:1]: remove version import

File: benchmark_tests/test_train_uvec_3d/uvec_ten_dof_vehicle_2D/__version__.py
Comment (aronnoordam) [inline benchmark_tests/test_train_uvec_3d/uvec_ten_dof_vehicle_2D/__version__.py:1]: remove version file

File: benchmark_tests/test_uvec_on_soil_3d/test_uvec_on_soil_3d.py [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+from stem.model import Model
+from stem.soil_material import OnePhaseSoil, LinearElasticSoil, SoilMaterial, SaturatedLaw
+from stem.load import UvecLoad
+from stem.boundary import DisplacementConstraint
+from stem.solver import AnalysisType, SolutionType, TimeIntegration, DisplacementConvergenceCriteria, StressInitialisationType, SolverSettings, Problem
+from stem.output import NodalOutput, VtkOutputParameters, Output
+from stem.stem import Stem
+
+import sys
+sys.path.append("./benchmark_tests")
```
Comment (aronnoordam) [inline benchmark_tests/test_uvec_on_soil_3d/test_uvec_on_soil_3d.py:13]:
```suggestion
```

File: benchmark_tests/test_uvec_on_soil_3d/test_uvec_on_soil_3d.py [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+from utils import assert_files_equal
```
Comment (aronnoordam) [inline benchmark_tests/test_uvec_on_soil_3d/test_uvec_on_soil_3d.py:14]:
```suggestion
from benchmark_tests.utils import assert_files_equal
```

File: benchmark_tests/test_uvec_on_soil_3d/test_uvec_on_soil_3d.py [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+    time_integration = TimeIntegration(start_time=0.0, end_time=1.0, delta_time=0.01, reduction_factor=1.0,
+                                       increase_factor=1.0, max_delta_time_factor=1000)
+
+    convergence_criterion = DisplacementConvergenceCriteria(displacement_relative_tolerance=1.0e-4,
+                                                            displacement_absolute_tolerance=1.0e-12)
+    stress_initialisation_type = StressInitialisationType.NONE
+    solver_settings = SolverSettings(analysis_type=analysis_type, solution_type=solution_type,
+                                    stress_initialisation_type=stress_initialisation_type,
+                                    time_integration=time_integration,
+                                    is_stiffness_matrix_constant=False, are_mass_and_damping_constant=False,
```
Comment (aronnoordam) [inline benchmark_tests/test_uvec_on_soil_3d/test_uvec_on_soil_3d.py:92]:
```suggestion
                                    is_stiffness_matrix_constant=True, are_mass_and_damping_constant=True,
```

**Review by brunozc — APPROVED** [review]

**Pull request merged**